cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -208,7 +227,9 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
                     }
                 }
 
-                val packageDownloadDirs = packages.associateWith { outputDir.resolve(it.id.toPath()) }
+                val packageDownloadDirs = packages
+                    .filter { isPackageLicenseCategoryIncluded(it, ortResult) }
```

Comment:
I have several ideas how to improve this function and the way it is called. But I propose to do this in a pair programming session.

Reply:
Done during pair programming.